packages/pn-pa-webapp/src/redux/apiKeys/actions.ts

```diff
@@ -54,7 +54,11 @@ export const newApiKey = createAsyncThunk<NewApiKeyResponse, NewApiKeyRequest>(
 
 export const getApiKeyUserGroups = createAsyncThunk<Array<UserGroup>, GroupStatus | undefined>(
   API_KEYS_ACTIONS.GET_API_KEY_USER_GROUPS,
-  performThunkAction((status: GroupStatus | undefined) => NotificationsApi.getUserGroups(status))
+  performThunkAction(async (status: GroupStatus | undefined) => {
```

Comment:
remove the use of performThunkAction. check the newApiKey action above to understand how to do